.github/workflows/publish-new-release.yml

```diff
@@ -0,0 +1,121 @@
+          CONVENTIONAL_GITHUB_RELEASER_TOKEN: ${{ secrets.PAT }}
+        run: |
+          git fetch --tags origin
+          git tag -a v${{ steps.extract-version.outputs.release_version }} -m "chore: release v${{ steps.extract-version.outputs.release_version }}"
+          git push origin refs/tags/v${{ steps.extract-version.outputs.release_version }}
+          npm run release:github
+          echo "DATE=$(date)" >> $GITHUB_ENV
+
+      - name: Pull Changes Into develop Branch
+        uses: repo-sync/pull-request@v2.12.1
+        with:
+          source_branch: 'master'
+          destination_branch: 'develop'
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          pr_title: 'chore(release): pull main into develop post release v${{ steps.extract-version.outputs.release_version }}'
+          pr_body: ':crown: *An automated PR*'
+
+      - name: Delete Release Branch
+        uses: koj-co/delete-merged-action@master
+        if: startsWith(github.event.pull_request.head.ref, 'release/')
+        with:
+          branches: 'release/*'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Delete Hotfix Release Branch
+        uses: koj-co/delete-merged-action@master
+        if: startsWith(github.event.pull_request.head.ref, 'hotfix-release/')
+        with:
+          branches: 'hotfix-release/*'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Notify Slack Channel
+        id: slack
+        uses: slackapi/slack-github-action@v1.23.0
+        continue-on-error: true
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+          PROJECT_NAME: 'Rudder Shopify Tracker'
+          RELEASES_URL: 'https://github.com/rudderlabs/rudder-shopify-tracker/releases/tag/'
+        with:
+          channel-id: ${{ secrets.SLACK_RELEASE_CHANNEL_ID }}
+          payload: |
+            {
+              "text": "*<${{env.RELEASES_URL}}v${{ steps.extract-version.outputs.release_version }}|v${{ steps.extract-version.outputs.release_version }}>*\nCC: <@U03KG4BK1L1> <@U02AE5GMMHV> <@U01LVJ30QEB> <@U03NAD30MTJ> <@U02AGPP420Y> <@U035SSC9NSH>",
+              "blocks": [
+                {
+                  "type": "header",
+                  "text": {
+                    "type": "plain_text",
+                    "text": ":tada:  ${{ env.PROJECT_NAME }} - New GitHub Release  :tada:"
+                  }
+                },
+                {
+                  "type": "divider"
+                },
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "*<${{env.RELEASES_URL}}v${{ steps.extract-version.outputs.release_version }}|v${{ steps.extract-version.outputs.release_version }}>*\nCC: <@U03KG4BK1L1> <@U02AE5GMMHV> <@U01LVJ30QEB> <@U03NAD30MTJ> <@U02AGPP420Y> <@U035SSC9NSH>"
+                  }
+                }
+              ]
```

Comment:
At the end of docker build push to repository, raise PR in devops repo for updating new tracker version

Reply:
added